integration/templates/astro-node/src/components/CustomUserButton.astro

```diff
@@ -0,0 +1,21 @@
+---
+import { UserButton } from '@clerk/astro/components';
+---
+
+<UserButton afterSignOutUrl="/">
```

Comment:
Can we also test here that https://clerk.com/docs/customization/user-button#reorder-default-items (e.g. the default sign-out) is working?

Reply:
Test added 👍🏼